blinkist-airbrake-scrubber.gemspec

```diff
@@ -8,15 +9,15 @@ Gem::Specification.new do |gem|
   gem.version       = Blinkist::AirbrakeScrubber::VERSION
   gem.authors       = ["Paweł Komarnicki", "Dinesh Vasudevan", 'Tomek Przedmojski']
```

Comment:
While you are at it, do you think we should also change Pawel to `Blinkist` and the email to operations@blinkist.com?